nuxeo-distribution/nuxeo-distribution-cap-webdriver-tests/pom.xml

```diff
@@ -168,6 +168,30 @@
     </profile>
     <profile>
       <id>jsfoptims</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <argLine>-Dlog4j.configuration=${file.uri.prefix}${basedir}/src/main/resources/log4j.xml
+                    -Dnuxeo.log.dir=${project.build.directory}
+                    -DconnectPassword=${connect.password}
+                    -Dnuxeo.jsf.tests.optimsEnabled=true
```

Comment:
Shouldn't it be parameterised ?

Reply:
Do you mean to avoid duplicating the conf, setting this specific parameter in the "jsfoptims" profile?

Reply:
:+1: done with 5123185147b242896d5f81057cee7c0da744fd8a (and similarly on https://github.com/nuxeo/nuxeo-platform-forms-layout-demo/commit/13eb0cecc44e48180de934aa02a02bc96c86b70c)